calligraphy/src/main/java/uk/co/chrisjenx/calligraphy/CalligraphyFactory.java

```diff
@@ -237,7 +237,9 @@ private ToolbarLayoutListener(final CalligraphyFactory calligraphyFactory,
                 }
             }
             removeSelf(toolbar);
-            toolbar.setSubtitle(originalSubTitle);
+            if (TextUtils.equals(toolbar.getSubtitle(), BLANK)) {
```

Comment:
I tried that and it didn't seem to behave differently. If users set subtitle to `" "` it would still reset it. I'm not entirely sure why though...